watchtower/src/main.rs

```diff
@@ -139,7 +140,15 @@ fn get_config() -> Config {
             Arg::with_name("monitor_active_stake")
                 .long("monitor-active-stake")
                 .takes_value(false)
-                .help("Alert when the current stake for the cluster drops below 80%"),
+                .help("Alert when the current stake for the cluster drops below the amount specified by --active-stake-alert-threshold"),
+        )
+        .arg(
+            Arg::with_name("active_stake_alert_threshold")
+                .long("active-stake-alert-threshold")
+                .value_name("PERCENTAGE")
+                .takes_value(true)
+                .default_value("80")
+                .help("Alert when the current stake for the cluster drops below this value"),
```

Comment:
Oh, sorry can you also please add a ` .validator(is_valid_percentage)` to this `Arg`

refs:
* https://github.com/solana-labs/solana/blob/f7b34e46ce5add296db002f74d779da1a6b09a7e/genesis/src/main.rs#L13
* https://github.com/solana-labs/solana/blob/f7b34e46ce5add296db002f74d779da1a6b09a7e/genesis/src/main.rs#L271

Reply:
done

Reply:
Pretty sure you'll need to import it too (the first ref) for the code to compile

Reply:
oof, thought I'd seen that before but was in a diff cli section. fixed

Reply:
indeed. done